music21/romanText/writeRoman.py

```diff
@@ -0,0 +1,536 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+# Name:         romanText/writeRoman.py
+# Purpose:      Writer for the 'RomanText' format
+#
+# Authors:      Mark Gotham
+#
+# Copyright:    Copyright © 2020 Michael Scott Cuthbert and the music21 Project
+# License:      BSD, see license.txt
+# ------------------------------------------------------------------------------
+'''
+Writer for the 'RomanText' format (Tymoczko, Gotham, Cuthbert, & Ariza ISMIR 2019)
+'''
+
+import fractions
+import unittest
+
+from typing import Optional, Union
+
+from music21 import base
+from music21 import metadata
+from music21 import meter
+from music21 import prebase
+from music21 import roman
+from music21 import romanText
+from music21 import stream
+
+
+# ------------------------------------------------------------------------------
+
+class RnWriter(prebase.ProtoM21Object):
+    '''
+    Extracts the relevant information from a source
+    (usually a :class:`~music21.stream.Stream` of
+    :class:`~music21.roman.RomanNumeral` objects) for
+    writing to text files in the 'RomanText' format (rntxt).
+
+    Writing rntxt is handled externally in the
+    :meth:~music21.converter.subConverters.WriteRoman so
```

Comment:
More backticks here and below needed. There are some instructions in the music21 developer's guide on building documentation, and if they aren't clear that's a brilliant opportunity for another PR. Here is what I see when I build (see the links not rendering):

<img width="800" alt="links-not-rendered" src="https://user-images.githubusercontent.com/38668450/99461353-ab430600-28ff-11eb-974f-c42b3882fc25.png">

Reply:
Sorry! Always proof-read last – no edits after the final checks.